### PR TITLE
[MM-46060] - Self-Managed: Add 'View Plan' button in plan card

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
Add View plan button that opens the pricing modal when on self-hosted starter

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-46060

#### Screenshots
Screenshot 2022-08-10 at 14 43 59

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by pvev — APPROVED** [review]
LGTM, thanks @AGMETEOR :shipit:

**Comment by marianunez** [comment]
Added @yasserfaraazkhan here for QA Review. 

@AGMETEOR Any E2E tests we could add here?

**Review by yasserfaraazkhan — APPROVED** [review]

**Comment by AGMETEOR** [comment]
> Added @yasserfaraazkhan here for QA Review.
> 
> @AGMETEOR Any E2E tests we could add here?

I have added an e2e test

File: e2e/cypress/tests/integration/enterprise/system_console/about/starter_edition_spec.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+// Group: @enterprise @not_cloud @system_console @license_removal
+
+describe('System console', () => {
+    before(() => {
+        // * Ensure we are on self-hosted Starter edition
+        cy.shouldNotRunOnCloudEdition();
+        cy.apiDeleteLicense();
+    });
+
+    it('License page shows View plans button', () => {
```
Comment (AGMETEOR) [inline e2e/cypress/tests/integration/enterprise/system_console/about/starter_edition_spec.js:20]: @yasserfaraazkhan we shall need to map this to zephyr

Comment (yasserfaraazkhan) [reply e2e/cypress/tests/integration/enterprise/system_console/about/starter_edition_spec.js:20]: Thank you Allan
I've added it to task: MM-45931

Comment (AGMETEOR) [reply e2e/cypress/tests/integration/enterprise/system_console/about/starter_edition_spec.js:20]: Alright. Thanks a lot. Going to merge this.

Comment (yasserfaraazkhan) [reply e2e/cypress/tests/integration/enterprise/system_console/about/starter_edition_spec.js:20]: @AGMETEOR

added this test in zephyr. Will update the ID with one of the PR.

 [MM-T5132](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/MM-T5132)

**Pull request merged**